src/sentry/api/event_search.py

```diff
@@ -1431,6 +1442,78 @@ def has_default(self, params):
         "transform": u"abs(minus({column}, {target:g}))",
         "result_type": "duration",
     },
+    # These range functions for performance trends, these aren't If functions
+    # to avoid allowing arbitrary if statements
+    # Not yet supported in Discover, and shouldn't be added to fields.tsx
+    "percentile_range": {
+        "name": "percentile_range",
+        "args": [
+            DurationColumn("column"),
+            NumberRange("percentile", 0, 1),
+            DateArg("start"),
+            DateArg("end"),
+            NumberRange("index", 1, None),
+        ],
+        "aggregate": [
```

Comment:
I am very confused about how this works. The `timestamp` column doesn't exist in the transaction table, and you don't actually pass `timestamp` in the groupby or or selected columns anywhere.

Reply:
So afaict the `timestamp` works since snuba aliases `finish_ts as timestamp` 
eg. locally this query:
```
{
    'consistent': False, 'having': [], 'aggregations': [[u'quantile(0.95)', 'duration', u'p95']], 'dataset': 'discover', 'project': [2L, 3L], 'from_date': '2020-08-19T02:02:02', 'limit': 51, 'to_date':'2020-08-20T02:02:02', 'offset': 0, 'conditions': [['type', '=', u'transaction'], ('project_id', 'IN', [3L, 2L])],
    'selected_columns': ['timestamp'], 'groupby': ['timestamp']
}
```
generates this sql:
```
SELECT
    finish_ts AS timestamp,
    quantile(0.95)(duration) AS p95,
    timestamp
FROM transactions_local
WHERE (finish_ts >= toDateTime('2020-08-19T01:58:02', 'Universal')) AND (finish_ts < toDateTime('2020-08-20T01:58:02', 'Universal')) AND (project_id
IN (2, 3)) AND (('transaction' AS type) = 'transaction') AND (project_id IN (3, 2))
GROUP BY timestamp
LIMIT 0, 51
```